utils/showcase.bash

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+# Setup a test workspace. We gitignore this.
+rm -rf showcase-testdir
+mkdir showcase-testdir/
+
+pushd showcase-testdir
```

Comment:
Why not just cd?

Reply:
Sourcing the script will probably mess up the caller's TRAP, if they have any. Added check to prevent sourcing.